Cargo.toml

```diff
@@ -31,18 +32,19 @@ futures = "0.3"
 futures-util = "0.3"
 git-version = "0.3.5"
 hex = "0.4.3"
-iroh-bitswap = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-metrics = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-rpc-client = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-rpc-types = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-store = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-util = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
+iroh-bitswap = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-metrics = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-rpc-client = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-rpc-types = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-store = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-util = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
 lazy_static = "1.4"
 libipld = "0.15"                                                                          # use same version as Iroh
 libp2p = { version = "0.51", default-features = false }                                   # use same version as Iroh
 libp2p-identity = { version = "0.1.2", features = ["ed25519"] }
 libp2p-tls = { version = "0.1.0", default-features = false }                              # use explicit version of dep to avoid conflict
 lru = "0.8"
+minicbor = { version = "0.19.1", features = ["alloc"] }
```

Comment:
Curious why the choice of minicbor? I have a WIP PR #61 where I expand our usage of cbor. Still basic usage. I can update it after this PR merges, but I'd like to understand what the other options where and why this one was selected.